src/tree/box.js

```diff
@@ -101,6 +102,16 @@ export default function box(
         selections.select(node, selectionsAndTransform, selectionState);
       }
     })
+    .each((node, index, cards) => {
+      Touche(cards[index]).tap({
+        end: () => {
+          if (!selectionsAndTransform.constraints.active && node.data.id !== 'Root') {
+            touchmode && openTooltip(tooltip, node, element.clientHeight, cardStyling, x, y, selectionsAndTransform, 0);
+            selections.select(node, selectionsAndTransform, selectionState);
+          }
+        },
+      });
```

Comment:
This call returns an unbind function, make sure to run those to remove listeners. I think doing this for each node is ok, as touch only registers a single touchevent on window, but best keep an eye on it.

Reply:
The Touche() call? Where can I find that?

Reply:
hmm, maybe it doesn't. just looked like it in the client, but that one wasn't used.  You can do:
`Touche(item.container).off('*', 'namespace');`
for a:
```
Touche(cards[index]).tap({
    id: 'namespace',
    end: () => {
       ...
       }
     },
   });
```

Reply:
Ah that should work yes. When do we want to clean them up? I guess the only place would be then in preRenderTree before removing the whole innerHTML, but wouldn't  the listeners be removed then anyway?

Reply:
I guess they would yea, but best the thorough on these things. Suddenly we stop removing the elements and are stuck with multiple listeners.

Reply:
Had to create a class for the elements to be removed. You need to have the specific element to remove the listener from it